src/components/AdminCreateTaskQueue.tsx

```diff
@@ -94,6 +106,23 @@ const AdminCreateTaskQueue = ({ setShowModal, handleDataChange}: IProps) => {
     setShowModal(false);
   };
 
+  const createMultiSelectOptions = () => {
+    var arr: Options[] = []
```

Comment:
Found more vars

Reply:
One of them causes an error when changing to let, but changed the others

Reply:
nvm all good